src/main/java/ru/hogwarts/school/model/Faculty.java

```diff
@@ -1,69 +1,41 @@
 package ru.hogwarts.school.model;
 
-import lombok.AllArgsConstructor;
-import lombok.EqualsAndHashCode;
-import lombok.Getter;
-import lombok.Setter;
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import jakarta.persistence.*;
+import lombok.*;
+
+import java.util.List;
 
-import java.util.Objects;
 @Getter
 @EqualsAndHashCode
 @Setter
 @AllArgsConstructor
+@ToString
+@Entity
 public class Faculty {
+   @Id
+   @GeneratedValue(strategy = GenerationType.IDENTITY)
     private long id;
     private String name, color;
 
     public Faculty() {
     }
 
-    @Override
-    public boolean equals(Object o) {
-        if (this == o)
-            return true;
-        if (o == null || getClass() != o.getClass())
-            return false;
-        Faculty faculty = (Faculty) o;
-        return id == faculty.id && Objects.equals(name, faculty.name) && Objects.equals(color, faculty.color);
-    }
-
-    @Override
-    public int hashCode() {
-        return Objects.hash(id, name, color);
-    }
-
-    @Override
-    public String toString() {
-        return "Faculty{" +
-                "id=" + id +
-                ", name='" + name + '\'' +
-                ", color='" + color + '\'' +
-                '}';
-    }
+    @OneToMany(mappedBy = "faculty")
+    @JsonIgnore
+    private List<Student> students;
 
-    public long getId() {
-        return id;
+    public Faculty(String faculty, String белый) {
     }
```

Comment:
Нужно поправить конструктор
```suggestion
 public Faculty(String name, String color) {
  this.name = name;
  this.color = color;
 }
```